localnode/endpoints.env

```
@@ -0,0 +1,4 @@
+BITCOINENDPOINT=
+BITCOINCREDS=
+GETHL1ENDPOINT=
+PRYSMENDPOINT=
```

nit: missing newline at end of file
```suggestion
PRYSMENDPOINT=

```

---

I HATE when files lack a newline at the end.  I have both `emacs` and `mg` set to ask me about it when saving.  No idea how I missed that.